draft-ietf-masque-connect-ip.md

```diff
@@ -1055,10 +1084,9 @@ URI suffix in the "Well-Known URIs" registry maintained at
 IANA is requested to update the "Reference" field to include this
 document in addition to previous values from that field.
 
-IANA is requested to add the following sentence to the "Related Information"
-field:
-: Includes all resources identified with the path prefix "/.well-known/masque/ip/".
-{: spacing="compact"}
+IANA is requested to replace the "Related Information" field with
+"See registry at IANA_URL_TBD." where IANA_URL_TBD is the URL of the new
```

Comment:
I would suggest "See registry for sub suffix allocations at IANA_URL_TBD."

Reply:
Agreed, tweaked the text